src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -58,8 +58,8 @@ public function __construct()
 
         $this->removingPasses = array(
             new RemovePrivateAliasesPass(),
-            new RemoveAbstractDefinitionsPass(),
             new ReplaceAliasByActualDefinitionPass(),
+            new RemoveAbstractDefinitionsPass(),
```

Comment:
This looks like a bug fix to me: aliases should be resolved before removing abstract definitions, isn't it?